src/components/video-player/VideoPlayer.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classes from './VideoPlayer.module.css';
+import { VideoPlayerFooter } from './VideoPlayerFooter';
+
+const VideoPlayer = ({ video = {} }) => {
+  const {
+    youtubeId,
+    channelImage,
+    title,
+    views,
+    likes,
+    uploadDate = 1648333768273,
+    channelName,
+    description,
+  } = video;
```

Comment:
It might be a personal opinion of mine but I guess it would be better if you destructure your arguments at the top only but the way you have written, this also works.

Reply:
Thanks for pointing it out. i feel what you have suggested is a good approach, also saves a few lines of codes.